buildpacks/anaconda.go

```diff
@@ -0,0 +1,127 @@
+package buildpacks
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	. "github.com/yourbase/yb/plumbing"
+	. "github.com/yourbase/yb/types"
+)
+
+type AnacondaBuildTool struct {
+	BuildTool
+	version string
+	spec    BuildToolSpec
+}
+
+// Fixed for Python3, later we'll create some support for Python2
+var ANACONDA_DIST_MIRROR = "https://repo.continuum.io/miniconda/Miniconda3-{{.Version}}-Linux-x86_64.sh"
```

Comment:
Does this work for other systems or just Linux? If others can we make the URL template take those properties?

Reply:
I just forgot to move the variable up, I had it right. Fixing it...